.github/workflows/main.yml

```diff
@@ -57,6 +57,26 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+      - name: Push beta app images
+        if: ${{github.ref == 'refs/heads/develop'}}
+        run: |
+          echo ${{ secrets.PAT }} | docker login ghcr.io -u pascal-airy --password-stdin
+          release_targets=$(bazel query "filter("beta$", //...)" --output label)
+          for target in $release_targets; do
+            echo "Deploying $target"
+            bazel run $target
+          done
+
+      - name: Push release app images
+        if: ${{github.ref == 'refs/heads/main'}}
+        run: |
+          echo ${{ secrets.PAT }} | docker login ghcr.io -u pascal-airy --password-stdin
```

Comment:
and here

I'm thinking we deleted the release script (on my suggestion I know) but maybe we should put it back and reuse it in both actions (which effectively are the same thing and the only difference is the tag)